sympy/sets/fancysets.py

```diff
@@ -621,13 +619,14 @@ class ComplexRegion(Set):
     is_ComplexRegion = True
 
     def __new__(cls, sets, polar=False):
-        from sympy import symbols, Dummy
+        from sympy import symbols, Dummy, sympify
 
         x, y, r, theta = symbols('x, y, r, theta', cls=Dummy)
         I = S.ImaginaryUnit
+        polar = sympify(polar)
 
         # Rectangular Form
-        if polar is False:
+        if polar == False:
```

Comment:
I always find the different kinds of booleans confusing. What would happen if we had ``if not polar:`` here?